include/kwk/detail/sequence/prefilled.hpp

```diff
@@ -203,13 +236,6 @@ namespace kwk::__
       );
     }
 
-    // Static axis access
-    template<std::size_t N>
-    KWK_TRIVIAL constexpr auto axis() const noexcept
```

Comment:
why is it removed ?

Reply:
explained in the commit message: sort of i needed quite a bit more functionality regarding (static) axis access so i expanded/split this into more parts (plus wherever i could/got the chance changed things not to use constexpr functions but rather variable templates, should be faster to compile afaik) - sort of#2 to be discussed where to put all that

Reply:
OK